### ISSUE TITLE
Failed to connect

### ISSUE THREAD

**Issue by k375725** [unixpickle/kahoot-hack#70]
I get this error almost every time...

failed to connect: failed to create session: parse session challenge: invalid character '<' looking for beginning of value
exit status 1

**Comment by unixpickle** [comment]
might be a duplicate of #64.

**Comment by k375725** [comment]
Yeah, just that it works with 2000 bots for like 4 minutes and then I get some of those errors.

**Comment by xenocidewiki** [comment]
Same issue, constantly getting:
failed to connect: websocket: bad handshake
exit status 1

**Comment by k375725** [comment]
Now it happens all the time after like 3-4 minutes...

**Comment by JasjarJasjar** [comment]
I have the same issue as xenocidewiki

**Comment by msemple1111** [comment]
Sometime within the last few months kahoot sometimes started sending a 'denied_by_security_policy' error. Perhaps this is the cause of the issue?
`[{'error': '403:denied_by_security_policy:create denied', 'id': '1', 'successful': False, 'channel': '/meta/controller', 'ext': {'timesync': {'ts': 1484256405060, 'a': -44, 'p': 0, 'tc': 1484256405030}}}]`

**Comment by msemple1111** [comment]
The error doesn't actually effect anything, only throws an error.

**Comment by unixpickle** [comment]
This is likely the result of a firewall.
